contributing/development/core_and_modules/object_class.rst

```diff
@@ -63,7 +63,7 @@ Object-derived classes can override the static function
 ``static void _bind_methods()``. When one class is registered, this
 static function is called to register all the object methods,
 properties, constants, etc. It's only called once. If an Object derived
-class is instanced but has not been registered, it will be registered as
+class is instanced, but has not been registered, it will be registered as
```

Comment:
```suggestion
class is instanced, but has not been registered it will be registered as
```

Reply:
I'd keep the comma before `it` but remove the one before `but`.

Reply:
This was me starting at commas too long and I removed the wrong one. I'll update this. :smile: